pandas/tests/io/formats/test_info.py

```diff
@@ -87,7 +101,7 @@ def test_info_verbose():
     frame.info(verbose=True, buf=buf)
 
     res = buf.getvalue()
-    header = " #    Column  Dtype  \n---   ------  -----  "
+    header = " #     Column  Dtype  \n---    ------  -----  "
```

Comment:
why has this changed?

Reply:
I changed this test because of the inconsistency with the column spacing I observed.
See "Note: needed to change test behavior. Here is the reason why." in the PR description.

Reply:
The spacing between the columns will not change in general. This particular test has 10000 columns, which makes the first column wide. As a result, there is an inconsistent spacing between the columns. I forced it to be 2 spaces regardless of the column width as I suppose it is reasonable.

Reply:
> See "Note: needed to change test behavior. Here is the reason why." in the PR description.

Thanks, I should've noticed that. I think your explanation and reasoning makes sense, though perhaps it would be better to have a separate PR to fix this, and then keep this one to enable polymorphism?

Reply:
OK, I will create the issue and make a small PR to fix it. But the present PR will basically overwrite this fix anyway.

Reply:
Created PR #36766.